docs/user/training_a_policy.md

```diff
@@ -0,0 +1,189 @@
+# Training a policy to solve an environment
+
+This page will guide you training a policy to solve an environment.
+
+## Define an experiment
+
+In garage, we train a policy in an experiment, which is a function wrapped by a
+decorator called `wrap_experiment`. Below is an simple example.
+`wrap_experiment` could have some arguments. You can see the [experiments doc](experiments)
+for details of running experiments.
+
+```py
+@wrap_experiment
+def my_first_experiment():
+    ...
+```
+
+## Construct a LocalRunner
+
+Within the experiment, we need a `LocalRunner` to sets up important state (such
+as a TensorFlow Session) for training a policy. To construct a `LocalRunner`, an
+experiment context called `ctxt` is needed. This is used to create the
+snapshotter, and we can set it `None` here to make it simple.
+
+Garage supports both PyTorch and TensorFlow. If you use TensorFlow, you should
+use `LocalTFRunner`.
+
+Besides, in order to produce determinism, you can set a seed for the random
+number generator.
+
+```py
+@wrap_experiment
+def my_first_experiment(ctxt=None, seed=1):
+    set_seed(seed)
+    # PyTorch
+    runner = LocalRunner(ctxt)
+    ...
+    # TensorFlow
+    with LocalTFRunner(ctxt) as runner:
+        ...
+```
+
+## Construct an environment
+
+Garage supports many environments. You can also implement your own environment
+like [this](implement_env). In this example, we choose `CartPole-V1`
+environment.
+
+```py
+env = GarageEnv(env_name='CartPole-v1')
+```
+
+## Construct a policy and an algorithm
+
+Construct your policy and choose an algorithm to train it. Here, we use
+`CategoricalMLPPolicy` and `TRPO`, you can also implement your own algorithm
+like [this](implement_algo). Your policy should be compatible with the
+environment's observations and action space (CNN for image observations,
+discrete policy for discrete action spaces, etc).
+
+```py
+policy = CategoricalMLPPolicy(name='policy',
+                              env_spec=env.spec,
+                              hidden_sizes=(32, 32))
+
+baseline = LinearFeatureBaseline(env_spec=env.spec)
+
+algo = TRPO(env_spec=env.spec,
+            policy=policy,
+            baseline=baseline,
+            max_path_length=100,
+            discount=0.99,
+            max_kl_step=0.01)
+```
+
+## Train your policy
```

Comment:
I think it's better to use a general, objective sub-header like 'Train the policy'.